.github/workflows/CI.yaml

```diff
@@ -68,6 +68,13 @@ jobs:
           args: --manifest-path tests/Cargo.toml --features sqlite
 
       - name: Install Diesel-CLI
+        if: ${{ matrix.rust != 'stable' }}
```

Comment:
I don't understand why this change is necessary

Reply:
If `matrix.rust` is not `stable`, `rustc` is an old version.
So the latest `diesel-cli` dependency packages cannot be compiled.
e.g. serde_spanned@v0.6.5 needs rustc 1.67 or newer

Then, I changed it as follows.
If `matrix.rust` is `stable`, use the latest dielse-cli.
If it is not `stable`, use the older version of diesel-cli.

Reply:
Hmm I guess I'd want to just bump the MSRV then, but no strong opinion.

Reply:
If I may, I would like to bump the MSRV.
However, I did this because I thought there might be some reason to keep the MSRV at 1.65.

Reply:
I think it's fine to bump it: https://github.com/diesel-rs/diesel/blob/2.1.x/rust-toolchain is at 1.68.0
:)

Reply:
Thank you for your advice.
I will bump the MSRV to 1.68.

Reply:
I tried to install diesel-cli@2.1.1 with rust:1.68.0.
Then it failed to compile.

```shell
❯ docker run --rm -it rust:1.68.0 cargo install --features postgres --no-default-features diesel_cli
    Updating crates.io index
  Downloaded diesel_cli v2.1.1
  Downloaded 1 crate (53.8 KB) in 0.25s
  Installing diesel_cli v2.1.1
  Downloaded anstyle-query v1.0.2
  Downloaded autocfg v1.1.0
  Downloaded anstyle-parse v0.2.3
  Downloaded anstream v0.6.7
  Downloaded aho-corasick v1.1.2
  Downloaded unicode-normalization v0.1.22
  Downloaded unicode-bidi v0.3.14
  Downloaded diesel_migrations v2.1.0
  Downloaded serde v1.0.195
  Downloaded anstyle v1.0.4
  Downloaded unicode-ident v1.0.12
  Downloaded overload v0.1.1
  Downloaded clap_complete v4.4.6
  Downloaded tinyvec_macros v0.1.1
  Downloaded dotenvy v0.15.7
  Downloaded byteorder v1.5.0
  Downloaded toml_datetime v0.6.5
  Downloaded memchr v2.7.1
  Downloaded diffy v0.3.0
  Downloaded iana-time-zone v0.1.59
  Downloaded bitflags v2.4.1
  Downloaded pq-sys v0.4.8
  Downloaded itoa v1.0.10
  Downloaded form_urlencoded v1.2.1
  Downloaded diesel_table_macro_syntax v0.1.0
  Downloaded equivalent v1.0.1
  Downloaded migrations_internals v2.1.0
  Downloaded colorchoice v1.0.0
  Downloaded serde_spanned v0.6.5
  Downloaded utf8parse v0.2.1
  Downloaded migrations_macros v2.1.0
  Downloaded serde_regex v1.1.0
  Downloaded clap_lex v0.6.0
  Downloaded heck v0.4.1
  Downloaded strsim v0.10.0
  Downloaded nu-ansi-term v0.46.0
  Downloaded quote v1.0.35
  Downloaded percent-encoding v2.3.1
  Downloaded diesel_derives v2.1.2
  Downloaded tinyvec v1.6.0
  Downloaded num-traits v0.2.17
  Downloaded proc-macro2 v1.0.76
  Downloaded indexmap v2.1.0
  Downloaded serde_derive v1.0.195
  Downloaded clap v4.4.16
  Downloaded toml v0.7.8
  Downloaded url v2.5.0
  Downloaded toml_edit v0.19.15
  Downloaded hashbrown v0.14.3
  Downloaded clap_builder v4.4.16
  Downloaded winnow v0.5.34
  Downloaded chrono v0.4.31
  Downloaded regex v1.10.2
  Downloaded syn v2.0.48
  Downloaded idna v0.5.0
  Downloaded regex-syntax v0.8.2
  Downloaded regex-automata v0.4.3
  Downloaded diesel v2.1.4
  Downloaded 58 crates (4.4 MB) in 0.45s
error: failed to compile `diesel_cli v2.1.1`, intermediate artifacts can be found at `/tmp/cargo-install7WzKFd`

Caused by:
  package `clap v4.4.16` cannot be built because it requires rustc 1.70.0 or newer, while the currently active rustc version is 1.68.0
  Try re-running cargo install with `--locked`
```

Could I bump to `1.70` ?
I was able to compile it with 1.70.

Reply:
No strong opinion here, I'm going to let @adwhit decide on this :)

Reply:
@adwhit What do you think about this topic?